server/tests/entity.rs

```diff
@@ -186,107 +186,222 @@ mod add_revision_mutation {
 }
 
 mod create_mutation {
+    use rstest::*;
     use server::uuid::EntityType;
     use test_utils::*;
 
+    #[rstest]
+    #[case(EntityType::Applet, Some(7), Option::<i32>::None)]
+    #[case(EntityType::Article, Some(7), Option::<i32>::None)]
+    #[case(EntityType::Course, Some(7), Option::<i32>::None)]
+    #[case(EntityType::CoursePage, Option::<i32>::None, Some(18514))]
+    #[case(EntityType::Event, Some(7), Option::<i32>::None)]
+    #[case(EntityType::Exercise, Some(7), Option::<i32>::None)]
+    #[case(EntityType::ExerciseGroup, Some(7), Option::<i32>::None)]
+    #[case(EntityType::GroupedExercise, Option::<i32>::None, Some(2217))]
+    #[case(EntityType::Solution, Option::<i32>::None, Some(4827))]
+    #[case(EntityType::Video, Some(7), Option::<i32>::None)]
     #[actix_rt::test]
-    async fn creates_entity() {
-        for entity in EntityTestWrapper::all().iter() {
```

Comment:
Finally we can get rid of this EntityTestWrapper. For that the parametrized tests are indeed useful. Please delete it also in the test utils.

Reply:
There are other tests still using EntityTestWrapper.